styles.css

```diff
@@ -0,0 +1,111 @@
+* {
+  box-sizing: border-box;
+
+}
+*,
+*before,
+*after {
+  box-sizing: inherit;
+
+}
+body {
+  margin: 0;
+  font-size: 18px;
+
+}
+.post {
+
+}
+.post_header {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: url('./images/sq.jpeg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  width: 50%;
+  color: #FFF;
+  text-shadow:
+    -1px -1px 0 #000,
+    1px -1px 0 #000,
+    -1px 1px 0 #000,
+    1px 1px 0 #000;  
+
+}
+.post_header > h1 {
+  font-size: 47px;
+  margin-top: 5px;
+}
+.post_header img {
+  height: 40px;
+  width: 40px;
+}
+/*trying to align the name and sprite*/
+.post_header > div > p {
+  display: inline-block;
+  text-align: center;
+
+}
+.post_content {
+  background-color: #CCEBFF;
+  width: 50%;
+  height: 100%;
+  margin-left: 50%;
+  padding: 200px 40px 40px 40px;
```

Comment:
If you define the right padding one, then you don't need to define the one for the left (if they are the same).
```css
padding: 200px 40px 40px;
```